scss/search.scss

```
@@ -5,13 +5,14 @@
   width: 84dvw;
   max-height: 88dvh;
   margin: 0 auto;
+  overflow-x: hidden;
   overflow-y: scroll;
 
   font-size: var.$base-font-size;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)